addons/l10n_vn/models/account_move.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_vn_edi_invoice_number = fields.Char(
+        string='VN: Invoice Number',
+        help='Invoice Number as appearing on SInvoice.',
```

Comment:
```suggestion
        help='Electronic Invoicing number.',
```
sInvoice is just one of the providers